### PR TITLE
Review

### PR DESCRIPTION
**Pull request by abdullah-asghar-khan**
Review of previous project

### CODE REVIEW

**Comment by abdullah-asghar-khan** [comment]
@Abdullah2213565  Please try adding some border-radius for head class in the code
170965851-b1955aa1-3ec3-4023-943c-f69aa0b898e9
.

**Comment by abdullah-asghar-khan** [comment]
Hi @Abdullah2213565  ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Required Changes ♻️
Check the comments under the review.

Cheers and Happy coding!👏👏👏

File: index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ <body>
+   <div class="main">
+     <div class="head">
+       <h1>Dr. Norman Borlaug</h1>
+       <p>The man who saved a billion lives</p>
+     </div>
+   <figure class = "images">
+       <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+       <figcaption>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+   </figure>
```
Comment (abdullah-asghar-khan) [inline index.html:18]: @Abdullah2213565 , this image is breaking the layout on some screens. I suggest you fix it.
* Please also add alt text for the image

Comment (danmainah) [reply index.html:18]: - [x] @Abdullah2213565 good review but I would suggest the use of polite words like please kindly when requesting a change, and also explain why the change needs to be done.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Abdullah2213565 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️  Good use of inline comments
✔️  Good design of your review summary.

# Required Changes ♻️

- [x] Kindly add an extra review comment in your review, you can use your previous comment which is not in your review, try adding it as an inline comment.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Munsa1 — APPROVED** [review]
Hi @Abdullah2213565 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Inline comments
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**